README.md

```diff
@@ -18,10 +18,9 @@ Currently the plugin supports [ClamAV anti-virus software](https://www.clamav.ne
 
 1. Go to the [releases page of this Github repository](https://github.com/mattermost/mattermost-plugin-antivirus/releases) and download the latest release for your Mattermost server.
 2. In the Mattermost System Console under **System Console > Plugins > Plugin Management** upload the file to install the plugin. To learn more about how to upload a plugin, [see the documentation](https://docs.mattermost.com/administration/plugins.html#plugin-uploads).
-3. Install ClamAV (clamd) for virus scanning. One easy option is to provision a ClamAV container with Docker by running the following command.  Assuming you have already installed Docker, this will download and install the latest version of ClamAV and set up a server with an open port at 3310:
-
+3. Install ClamAV (clamd) for virus scanning. One easy option is to provision a ClamAV container with Docker by running the following command.  Assuming you have already installed Docker, this will download and install the latest version of ClamAV and set up a server with an open port at 3310: and MaxFileSize of 50 MB. Mattermost's MaxFileSize default value is subject to change. To ensure that the correct value is set, verify your value at the following link: [Maximum File Size](https://docs.mattermost.com/administration/config-settings.html#maximum-file-size).  
    ```
-   docker run -d -p 3310:3310 mkodockx/docker-clamav
+   docker run -e CLAMD_CONF_MaxFileSize=50M -d -p 3310:3310 mkodockx/docker-clamav
    ```
 
 4. Once clamd server is running, configure the plugin to make requests to your clamd instance. Go to **System Console > Plugins > Antivirus** and configure **Clamav Host and Port** to point at your clamd instance.  
```

Comment:
@Francois-D I JUST ran intot his with someone else...it's not obvious what the clamAV hostname is that should be put into the "ClamAC Host and Port".  For example, someone put in "localhost:3310" to point to clamAV, but that won't work.  Would the default docker hostname for the ClamAV docker image be "docker-clamav:3310"?  If so, can we insert that as an example so it's clearer to admins on setup?

Reply:
@aaronrothschild 
The documentation should have a step 4 which checks if the docker container was launched correctly.  The command is _docker ps -a_

<img width="1240" alt="Screen Shot 2020-04-29 at 1 19 53 AM" src="https://user-images.githubusercontent.com/46973576/80563043-b6603400-89b7-11ea-8b44-c8e9d246e5c7.png">

The new step 5 would be to configured the **Clamav Host and Port**   ( A screenshot would aid in knowing where to look).  to point to the clamav instance.

Mattermost does not function when using 'localhost' in the clamav configuration.  In order to function the _hostname_ of the Docker host should be set. In my case the docker host was the same as the Mattermost Server.  To retrieve the hostname of the docker host run the command: _hostname -s_

As the configuration is pre populated with the value _localhost_:_port_.  Is it possible/makes sense  that value be set to the hostname of the server?  
<img width="929" alt="Screen Shot 2020-04-28 at 7 23 15 PM" src="https://user-images.githubusercontent.com/46973576/80563170-4605e280-89b8-11ea-8031-ece20be13c35.png">